device-types/Cisco/FPR3110-NGFW-K9.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: Cisco
+model: FirePOWER 3110 Next-Generation Firewall
+slug: cisco-firepower-3110-next-generation-firewall
+part_number: FPR3110-NGFW-K9
+u_height: 1
+is_full_depth: true
```

Comment:
Please add `weight` and `weight_unit` if you can

Reply:
```suggestion
is_full_depth: true
weight: 10.5
weight_unit: kg
```